.github/workflows/deploy-tobira-opencast.yml

```diff
@@ -0,0 +1,75 @@
+name: Deploy Tobira Opencast
+
+on:
+  push:
+    branches:
+      - main
+      # remove before merging
+      - tobira-deployment
+
+jobs:
+  build:
+    name: deploy tobira opencast
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: cache local maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
```

Comment:
Not sure if that's worth it. GH caches are also just downloaded from some server, so GitHub usually recommends against caching any stuff that "just" has to be downloaded from somewhere.

Or are the maven servers unreliable or slow?

Reply:
It's about 4 minutes (I hope I remember correctly) faster if we cache this.

Reply:
Ok yeah, then it seems fine.